### PR TITLE
issue #807 Issue templates

### PR DESCRIPTION
**Pull request by DVEfremov**
https://github.com/networkservicemesh/sdk/issues/807

### CODE REVIEW

**Review by denis-tingaikin — APPROVED** [review]
Looks good to me

**Comment by denis-tingaikin** [comment]
@DVEfremov Could you sign your commits to make CI green? Look at https://github.com/networkservicemesh/sdk/pull/826/checks?check_run_id=2331140245

**Comment by denis-tingaikin** [comment]
@edwarnicke Could you have a look at this PR?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Please switch to **Preview** and select one of the following links depending on your use-case:
+
+* [Feature](?template=spec.md)
+* [Bug](?template=bug.md)
+* [Question](?template=question.md)
```
Comment (edwarnicke) [inline .github/ISSUE_TEMPLATE.md:5]: I like the addition of Question here :)

**Review by edwarnicke — APPROVED** [review]

**Pull request merged**